### DISCUSSION TITLE
Can skywalking monitor applications under Weblogic

### DISCUSSION THREAD

**Discussion by POPIWEN0209** [apache/skywalking#8460] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Can skywalking monitor applications under Weblogic

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
No, it isn't. And in most cases(such as you are using SpringMVC), you don't need a plugin for Weblogic.
